modules/phase_field/include/utils/EBSDAccessFunctors.h

```diff
@@ -21,8 +21,8 @@ class EBSDAccessFunctors
 public:
   /// Per element EBSD data point
   struct EBSDPointData {
-    Real phi1, phi, phi2, symmetry;
-    unsigned int grain, phase, op, global;
+    Real phi1, Phi, phi2;
```

Comment:
No hostility, you already convinced me that this is "the normal way". I just wanted to know if there was a reference or some wiki page or something. If not, fine, just asking.